tests/Js/Main/MiscTests.fs

```diff
@@ -489,7 +489,12 @@ let tests =
         to_str () |> equal $"{()}"
 
 #if FABLE_COMPILER
-#if !FABLE_COMPILER_JAVASCRIPT
+#if !FABLE_COMPILER_TYPESCRIPT
+    // The tests seems to cause problems in TypeScript because of the typing
+    // system. We suppose this is probably due to the TypeProvider generation
+    // so for now we just ignore that situation hoping this is not
+    // indicative of other potential issues like this
+    // See: https://github.com/fable-compiler/Fable/pull/3748
```

Comment:
@MangelMaxime Looks like that test is failing for JS too

Reply:
This seems to be because I forgot to update the compiler directive for the `open` statement.

I have it working locally now, let's hope this is the final CI run and we can merge this PR.

Reply:
Looks like not, the test was disable for JavaScript because when testing using `fable-standalone` we don't support TypeProviders. 

But ok, that's not related to my PR changes will fix it with an additional compiler directive.